tests/unit/classifier/LDA_unittest.cc

```diff
@@ -233,6 +233,31 @@ TEST(LDA, DISABLED_CheckProjection_SVD)
 	EXPECT_NEAR(+7.96084156, projection_SVD[9], epsilon);
 }
 
+// label type exception test
+TEST(LDA, labels_type_exception)
+{
+	SGVector<float64_t> lab;
+	SGMatrix<float64_t> feat;
+
+	CRegressionLabels* labels;
+	CDenseFeatures<float64_t>* features;
+
+	generate_test_data<float64_t>(lab, feat);
+
+	labels = new CRegressionLabels(lab);
+	features = new CDenseFeatures<float64_t>(feat);
+
+	SG_REF(labels);
+	SG_REF(features);
+
+	CLDA lda_SVD(0, features, labels, SVD_LDA);
+
+	EXPECT_THROW(lda_SVD.train(), ShogunException);
```

Comment:
great!

few comments

* can we avoid all this redundant code via re-use the code from other tests?
* could you test other label types as well (like to test the multiclass conversion)?

Reply:
will it not be better to add the regressionlabels exception test in multiclasslabels unit test ?